typegate/src/runtimes/python_wasi/python_wasm_messenger.ts

```diff
@@ -14,7 +14,7 @@ export class PythonWasmMessenger extends AsyncMessenger<
   private constructor() {
     const vmMap = new Map<string, PythonVirtualMachine>();
     super(
-      vmMap,
+      () => vmMap,
```

Comment:
In the new start scheme introduced, after stopping a broker, we call `start(broker)`.  For python runtime's case, start has no effect since all VMs are initialized at init.